Event/DoctrineBackendOptimizeListener.php

```diff
@@ -30,9 +28,9 @@ class DoctrineBackendOptimizeListener implements IterationListener
     protected $doctrine;
 
     /**
-     * @param RegistryInterface $doctrine
+     * @param ContainerAwareInterface $doctrine
      */
-    public function __construct(RegistryInterface $doctrine)
+    public function __construct(ContainerAwareInterface $doctrine)
```

Comment:
I replace the type-hint of doctrine parameter as Doctrine\Common\Persistence\ManagerRegistry.